docs/api.md

```diff
@@ -1656,7 +1656,9 @@ await browser.close();
 
 Routing provides the capability to modify network requests that are made by a page.
 
-Once routing is enabled, every request matching the url pattern will stall unless it's continued, fulfilled or aborted.
+Once routing is enabled, every request* matching the url pattern will stall unless it's continued, fulfilled or aborted.
+
+_* The handler will only be called for the first url if the response is a redirect._
```

Comment:
We use notes for such things (look around in api.md for examples):

```> **NOTE** Only works with persistent context.```

Reply:
Updated